doc/conf.py

```diff
@@ -45,8 +45,11 @@
     "sphinx.ext.intersphinx",
     "sphinx_automodapi.automodapi",
     'sphinx_copybutton',
+    "m2r"
```

Comment:
I needed to install m2r to build the docs. Do we need to add it to requirements?

Reply:
Added it to `doc/requirements.txt`!